src/Build/Collections/RetrievableEntryHashSet/HashSet.cs

```diff
@@ -78,12 +77,12 @@ namespace Microsoft.Build.Collections
     /// the same time. 
     /// </summary>
     /// <typeparam name="T"></typeparam>
-    [DebuggerTypeProxy(typeof(Microsoft.Build.Collections.HashSetDebugView<>))]
+    [DebuggerTypeProxy(typeof(HashSetDebugView<>))]
```

Comment:
Please revert this file. It's copy-🍝, and these changes will make it hard to pull improvements to the base type in here.

Reply:
Point. Need to fix the original source then pull back. :) Kept one line of change (removal) pasted below since it's a perf win in a critical path:

 _slots[i].hashCode = hashCode;